.github/workflows/build-test-deploy.yml

```diff
@@ -477,14 +523,24 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Remove watcher from version tag
+        run: |
+          tag=${{ github.ref_name }}
+          if [[ $tag == *watcher:* ]]; then
+            new_tag=${tag/watcher:/}
+            echo "::set-output name=docker_tag::$new_tag"
+          else
+            echo "::set-output name=docker_tag::''"
+          fi
+
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
             type=sha,format=short
-            type=semver,pattern={{raw}}
+            type=raw,value=${{ steps.build-and-push-router-watcher-image.outputs.docker_tag }}
```

Comment:
This should instead be:
`type=raw,value=${{ steps.build-and-push-watcher-image.outputs.docker_tag }}`

Reply:
Nice catch! Fixed